deploy/doc/config.md

```diff
@@ -0,0 +1,69 @@
+[//]: # (This file is built out of deploy/doc/config.md, please do not edit it manually)  
+[//]: # (To rebuild it run `make config`)
+
+# Configuration
+
+## ClickHouse `[clickhouse]`
+
+### URL `url`
+Detailed explanation of ClickHouse HTTP interface is given in [documentation](https://clickhouse.tech/docs/en/interfaces/http). It's recommended to create a dedicated read-only used for graphite-clickhouse.
```

Comment:
`dedicated read-only used for` - not clear what exactly to create, I guess user?